### DISCUSSION TITLE
"Add to chrome" in Ecosia, can it go away?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7728] [Q&A] [answered]
Hello, I have been using Ecosia with qutebrowser, and it has worked great except that Ecosia thinks that I am on Chrome and on every search page asks me to add their extension.

I was wondering if anyone knew if there were any ways I could tell Ecosia not to make this popup through browser settings. I have contacted Ecosia and they said that they could not answer because they don't officially support an extension qutebrowser.

Thanks!

**Comment by The-Compiler** [comment, marked as answer]
You can drop a Greasemonkey userscript such as:

```js
// ==UserScript==
// @name         hide ecosia chrome button
// @include      https://www.ecosia.org/*
// @run-at       document-idle
// ==/UserScript==

document.getElementsByClassName("main-header__install-cta")[0].remove()
```

into `greasemonkey/ecosia.user.js` in your data or config directory (see `:version`).

**Reply by ghost** [reply]
Thank you, I completely forgot to look into greasemonkey.
